puppet/modules/slave/manifests/init.pp

```diff
@@ -150,12 +150,12 @@
     source => 'puppet:///modules/slave/JSON.sh',
   }
 
-  # nodejs/npm for Katello JavaScript tests
-  # packages only really available on EL6+
+  # nodejs/npm for JavaScript tests
   if $::osfamily == 'RedHat' {
-    package { 'npm':
-      ensure  => present,
-      require => Package['bzip2'],
+    class { 'nodejs':
+      repo_url_suffix       => '12.x',
+      nodejs_package_ensure => latest,
+      npm_package_ensure    => latest,
```

Comment:
needs to be `absent`, as the `nodejs` package is already containing it.

Reply:
Updated. Also explicitly set `manage_repo` to true.

Reply:
`Class[Nodejs]: has no parameter named 'manage_repo'` :upside_down_face: